bin/prettier.js

```diff
@@ -193,7 +193,11 @@ if (stdin) {
       if (write) {
         // Don't write the file if it won't change in order not to invalidate
         // mtime based caches.
-        if (output !== input) {
+        if (output === input) {
+          console.log(chalk.grey("%s %dms"), filename, Date.now() - start);
```

Comment:
When I wrote that comment I didn't realize you were printing the time taken for all files. You can ignore my comment and keep it.